ui/tract-renderer/src/main/java/org/cru/godtools/tract/ui/controller/FallbackController.kt

```diff
@@ -0,0 +1,17 @@
+package org.cru.godtools.tract.ui.controller
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import org.cru.godtools.tract.databinding.TractContentFallbackBinding
+import org.cru.godtools.xml.model.Fallback
+
+class FallbackController(
+    binding: TractContentFallbackBinding,
+    parentController: BaseController<*>?
+) : ParentController<Fallback>(Fallback::class, binding.root, parentController) {
+    internal constructor(parent: ViewGroup, parentViewHolder: BaseController<*>?) :
+        this(TractContentFallbackBinding.inflate(LayoutInflater.from(parent.context), parent, false), parentViewHolder)
+
+    override val contentContainer = binding.root
+    override val contentToRender get() = model?.content?.take(1)
+}
```

Comment:
So is this a fall back controller for all things? Like content, images, buttons, etc? That's pretty cool!

Reply:
the concept of the fallback content element (and this controller) is that it will render the first child element that is supported by the device. This will allow us to introduce animations and video to the content xml, but still provide "fallback" behavior for older versions of the app.